guide/14-deep-learning/object-detection.ipynb

```diff
@@ -6,7 +6,7 @@
    "source": [
```

Comment:
<p>"for consecutive 5 epochs" -&gt; "for 5 consecutive epochs"</p><p> </p><p>make "Note" bold.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/559/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>